3style-text/styles.css

```diff
@@ -0,0 +1,89 @@
+/* ---font styles--- */
+
+body {
+  font-family: "Public Sans", Verdana, sans-serif;
+  font-size: 18px;
```

Comment:
I'm not super familiar with fonts as percents. Usually I use `rem` but this is neat to look through! I may have to read the article to understand why `18px` is the default (usually when using `rem` the default is `16px`) 🤔

Reply:
I think REM is the preference--Zastrow demonstrates what each means in the tutorial. I love these tutorials!